### DISCUSSION TITLE
Contrato de serviço para freelancer

### DISCUSSION THREAD

**Discussion by mandado** [frontendbr/forum#142] [Discussão]
Fala galera!, tava pensando aqui. Vocês fazem contrato para os seus freelas ? 
Se sim:
- O que levam em consideração na hora de criar.
- O que acham interessante ter ou ideal no contrato.

Estou perguntando porque nunca fiz um contrato hehe, e acho que pode ajudar outras pessoas também.

**Comment by woliveiras** [comment]
Excelente pergunta cara. Eu usava um modelo que tinha [aqui](http://www.escolafreelancer.com/modelo-de-contrato-prestacao-de-servicos-freelancer/) quando fazia freelas.

Nem sei se é o mais adequado, só que assustava os clientes que vinham com o papo de:

_\- Mês que vem pago a parcela que era pra pagar nesse mês, tá?_

kkkkkk

**Comment by felipefialho** [comment]
Nunca fiz freelas com contrato, não sei responder =( 

@Mendrone, @vitorbritto e @eduardojmatos, sabem responder essa?

**Comment by mandado** [comment]
@woliveiras  eita! kkkkk, ainda não passei por situação.

@LFeh eu também nunca fiz, mas acho que seria uma boa começar a adotar pra evitar dor de cabeça, eu passei por uma dor de cabeça que foi tenso, mas foi a muito tempo hehe.

**Comment by vitorbritto** [comment]
@mandado já fiz freela sem contrato e algumas experiências foram traumáticas. Mas admito que existem casos e casos.

## Antes de tudo...

Quando uma empresa / agência ou um cliente vai em busca de um Freelancer, ao mesmo tempo ele vem atrás de um profissional com regras mais flexíveis (não são todos os casos, mas em geral é assim). 

Em uma agência, temos a emissão de Nota Fiscal, formalização de serviço por contrato, existe um horário de trabalho definido e outras regras a serem seguidas. 

O valor praticado em uma agência é diferenciado do que um freelancer pratica. Uma agência precisa honrar a hora técnica da sua equipe, suas despesas, impostos, entre outras obrigações.

Não seria muito diferente de um Freelancer, que também possui suas despesas, pode vir a terceirizar parceiros / mão de obra e tem horários. Entretanto, a imagem que os clientes possuem deste é que existem regras mais maleáveis e um valor mais em conta que uma agência aplicaria. Existe também o fato de que nem todo Freelancer segue uma metodologia de trabalho ou possui disciplina para seus afazeres.

## Ter ou não ter um contrato?

Não ter um contrato de prestação de serviço pode lhe trazer muita dor de cabeça, principalmente no que se refere ao quesito **entrega x pagamento**. Se você for terceirizado, ferrou mais ainda. O cara que terceirizou seu serviço ficará preso ao que o cliente dele decidir estar **ok** ou não. Enquanto isso você espera, espera, espera...

Acontece que não ter um contrato, não significa que você esteja equivocado. Temos situações onde há a confiança e o conhecimento prévio diante de um _job_ que foi executado. Um conhecimento, alinhamento, sincronização e entendimento a respeito da entrega, do pagamento, da garantia e dos **ajustes x alterações**.

É mais sensato emitir nota fiscal e mandar contrato por carta registrada via Correios. A NF serve para você declarar seu IR e o contrato para você se acobertar de qualquer dano causado pelo cliente e vice-versa.

IMHO, o cliente correto e que preza pelo compromisso não vai discutir com você sobre a questão de ter um contrato e executar o _job_ com Nota Fiscal. Isso é o de menos. O que ele vai analisar e  valorizar será o investimento e prazo / cronograma.

## Como proceder então?

Modelos são modelos. Contratos são documentos importantes e que uma vírgula pode mudar o sentido da sentença e te partir ao meio. Procure um advogado, que possa lhe orientar na montagem do seu contrato.

**Importante deixar claro no contrato:**
- O que será feito no serviço (mandar um escopo para ser aprovado pelo cliente)
- Suas obrigações e deveres
- As obrigações e deveres do cliente
- O cronograma a ser cumprido, com detalhes sobre as aprovações do cliente
- O investimento a ser pago, para quem será e como será feito
- A multa ou consequências pelo atraso no apagamento
- Observações gerais sobre serviços complementares, corte de versão, manual, garantia, etc...

Tem **muito** mais sobre o que falar. Esse é um assunto que daria 2 horas de _Hangout_ fácil.

Espero ter ajudado. =]

**Comment by mandado** [comment]
@vitorbritto  Não só me ajudou como acho que vai ajudar muita gente hehe, eu não tenho muito o que falar sobre isso, por falta de experiencia hehe, mas se quiser fazer um hangout, pra falar disso estou ai :D

**Comment by vitor-mendrone-deel** [comment]
Não tem nem o que complementar. Sensacional a resposta do @vitorbritto e vai ajudar muita gente!
